addons/l10n_dk/data/template/account.tax-dk.csv

```diff
@@ -63,12 +63,12 @@
 "","","","","","","","","","","tax","invoice","","","","",""
 "","","","","","","","","","","base","refund","","","","",""
 "","","","","","","","","","","tax","refund","","","","",""
-"tax_s4",20,"S-%EU-V-0","Sales of goods B2B (% EU), 0% VAT","0 %","0.0","percent","sale","False","True","base","invoice","","","","Varesalg B2B (% EU), 0% moms","0 %"
-"","","","","","","","","","","tax","invoice","dk_coa_7740","+MVU","","",""
-"","","","","","","","","","","tax","invoice","dk_coa_7700","-KM",-100,"",""
-"","","","","","","","","","","base","refund","","","","",""
-"","","","","","","","","","","tax","refund","dk_coa_7740","-MVU","","",""
-"","","","","","","","","","","tax","refund","dk_coa_7700","+KM",-100,"",""
+"tax_s4",20,"S-%EU-V-0","Sales of goods B2B (% EU), 0% VAT","0 %","0.0","percent","sale","False","True","base","invoice","","R-C-MR","","Varesalg B2B (% EU), 0% moms","0 %"
+"","","","","","","","","","","tax","invoice","dk_coa_7740","","","",""
+"","","","","","","","","","","tax","invoice","dk_coa_7700","",-100,"",""
+"","","","","","","","","","","base","refund","","-R-D-MR","","",""

```

Comment:
@malb-odoo Not entirely correct here. It should be `+R-C-UR` for `invoice` and `-R-C-UR` for `refund` as the tax code is for sales outside the EU

Reply:
missed the copy paste :facepalm: ahaha

Reply:
You have seen nothing :eyes: (just pushed the real fix ahah)